### PR TITLE
Fixes #20556: Add setup custom port 

### PR DESCRIPTION
**Pull request by junnhy5**
https://issues.rudder.io/issues/20556

### CODE REVIEW

File: roles/rudder_server/tasks/configure.yml [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-    group: root
-    mode: '0644'
+---
+- name: Configure listen ports for Rudder 6 
+  template:
+    src: rudder6-web.properties.j2
+    dest: "/opt/rudder/etc/rudder-web.properties"
+  when: server_version is version('6.2', operator='<=')
+
+- name: Configure listen ports for Rudder 7
```
[the remark this answers is not in the document]
Comment (julienbriault) [reply roles/rudder_server/tasks/configure.yml:10]: fixup !

File: roles/rudder_agent/tasks/configure.yml [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
     dest: "/var/rudder/cfengine-community/policy_server.dat"
     owner: root
     group: root
     mode: '0644'
+  when: "{{ agent_version is version('6.2', operator='<=') }}"
+
+- name: Define policy server for Rudder 7
+  ansible.builtin.copy:
+    content: "{{ policy_server }}:{{ policy_port }}"
+    dest: "/opt/rudder/etc/policy_server.dat"
```
Comment (Fdall) [inline roles/rudder_agent/tasks/configure.yml:14]: Why is it different in 7.0?

Comment (julienbriault) [reply roles/rudder_agent/tasks/configure.yml:14]: The destination file is not in the same place.

**Comment by Fdall** [comment]
Ass seen in the chat, remove the 6.2 part of the agent policy port, and add the configuration of the https_port on the agent for 7.0+ as described here https://docs.rudder.io/reference/7.0/administration/port.html#_set_connection_port_on_the_agent

**Pull request closed without merging**